pallets/capacity/src/lib.rs

```diff
@@ -278,6 +323,11 @@ impl<T: Config> Pallet<T> {
 		StakingAccountLedger::<T>::insert(staker, staking_account);
 	}
 
+	fn remove_staking_account(staker: &T::AccountId) {
```

Comment:
Nice! how about calling it" delete_" instead of "remove" to align with CRUD naming?

Reply:
legit